conan/tools/system/pip_manager.py

```diff
@@ -0,0 +1,64 @@
+import venv
+import platform
+import os
+
+from conan.errors import ConanException
+from conan.tools.env.environment import Environment
+
+
+class PipEnv:
+    accepted_install_codes = [0]
+    accepted_check_codes = [0, 1]
+
+    def __init__(self, conanfile, folder=None):
+        self._conanfile = conanfile
+        self.env_name = f"pip_venv_{self._conanfile.name}"
+        self._env_dir = os.path.abspath(os.path.join(folder or self._conanfile.recipe_folder, self.env_name))
+        self.bin_dir = os.path.join(self._env_dir, "Scripts" if platform.system() == "Windows" else "bin")
+        self._python_exe = os.path.join(self.bin_dir, "python.exe" if platform.system() == "Windows" else "python")
+
+    def _conanfile_run(self, command, accepted_returns):
+        ret = self._conanfile.run(command, ignore_errors=True, quiet=True)
+        if ret not in accepted_returns:
+            raise ConanException("Command '%s' failed" % command)
+        return ret
+
+    def _check(self, packages):
+        packages_to_install = []
+        if not os.path.exists(self._env_dir):
+            return packages
+        for package in packages:
+            name, version = (package.split("=")[0], package.split("=")[1]) if "=" in package else (package, "")
```

Comment:
I don't think this line is valid for python packages setting a specific version, because pip version specifiers use `==` and also more advanced expresions like: `>=`, `<=`. I personally think It would be safer to either rely on pip itself to validate/install the specifier and pass extra `cli_args` or something similar to the method?